aim/cli/runs/commands.py

```diff
@@ -0,0 +1,55 @@
+import click
+import os
+
+from aim.sdk.repo import Repo
+
+
+@click.group()
+@click.option('--repo', required=False,
+              default=os.getcwd(),
+              type=click.Path(
+                  exists=True,
+                  file_okay=False,
+                  dir_okay=True,
+                  writable=True
+              ))
+@click.pass_context
+def runs(ctx, repo):
+    """Manage runs in aim repository."""
+    ctx.ensure_object(dict)
+    ctx.obj['repo'] = repo
+
+
+@runs.command(name='ls')
+@click.pass_context
+def list_runs(ctx):
+    repo_path = ctx.obj['repo']
+    if not Repo.exists(repo_path):
+        click.echo(f'\'{repo_path}\' is not a valid aim repo.')
+        exit(1)
+    chunks_dir = os.path.join(repo_path, '.aim', 'meta', 'chunks')
+    run_hashes = os.listdir(chunks_dir)
+
+    click.echo('\t'.join(run_hashes))
+    click.echo(f'Total {len(run_hashes)} runs.')
+
+
+@runs.command(name='rm')
+@click.argument('run_hash', nargs=-1, type=str)
```

Comment:
`run_hash` or just `hash`? 🤔

Reply:
@gorarakelyan will change.
actually this is shown just in a help message. the argument is positional so no need to specify `--run-hash`/`--hash`

Reply:
UPD: @gorarakelyan cannot change, since the argument name should match the underlying command variable name. in that case `hash` will shadow python's built-in `hash()`.